assets/components/contribAmounts/contribAmounts.jsx

```diff
@@ -0,0 +1,173 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+
+import RadioToggle from 'components/radioToggle/radioToggle';
+import NumberInput from 'components/numberInput/numberInput';
+import { clickSubstituteKeyPressHandler } from 'helpers/utilities';
+import type { Contrib, ContribError, Amounts } from 'helpers/contributions';
+
+
+// ----- Setup ----- //
+
+const amountToggles = {
+  recurring: {
+    name: 'contributions-amount-recurring-toggle',
+    radios: [
+      {
+        value: '5',
+        text: '£5',
```

Comment:
Additionally, I would define types for these objects. It can help us to check everything with flow.